AnisotropyFilters/AdaptiveAlignment.cpp

```diff
@@ -188,7 +189,10 @@ void AdaptiveAlignment::dataCheck()
   tempPath.update(getDataContainerName(), getCellAttributeMatrixName(), "");
   getDataContainerArray()->getPrereqAttributeMatrixFromPath<AbstractFilter>(this, tempPath, -301);
 
-  FileSystemPathHelper::CheckOutputFile(this, "Alignment Shift File Name", getAlignmentShiftFileName(), true);
+  if(getWriteAlignmentShifts())
```

Comment:
No where in the filter does it actually use that filter parameter parameter so rather than confuse the user I simply commented out the ability to set the file. We didn't originally write the filter so I am unsure of the intention of the original author. I believe commenting out the ability is a conservative adjustment which keeps changes to a minimum.

@OndrejSedivy Comments are also welcome.